### PR TITLE
Feat: add the about page for the grommet docs site

### PR DESCRIPTION
**Pull request by RyanCCollins**
#### What does this PR do?
Adds an About page with info about the grommet team

#### Where should the reviewer start?
`src/docs/reference/About.js`

#### What testing has been done on this PR?
Manually tested by running in the browser.

#### How should this be manually tested?
Run in the browser and navigate to `http://localhost:8002/docs/about`

#### What are the relevant issues?
#162 

#### Screenshots (if appropriate)
grommet about page

### CODE REVIEW

**Comment by RyanCCollins** [comment]
Added @ericsoderberghp's twitter account and fixed the eslint issues.

File: src/docs/reference/About.js [hunk 1]
```diff
@@ -3,21 +3,131 @@
...
+          <Paragraph size="medium">
+            We originally created Grommet to be a solution 
+            for our own organization’s needs. We quickly 
+            found the usefullness for a tool like this outside 
+            Hewlett Packard, and taking a note from our own 
+            founders, Dave and Bill, we shared it with the 
+            open source community with great response. As 
+            with any company, change is inevitabe, 
+            and change it did…Hewlett Packard became 
+            two companies. The split became proof that G
```
Comment (alansouzati) [inline src/docs/reference/About.js]: I believe this a line break problem here? Not sure if it is affecting the actual page, but it seems odd to have "G" alone

File: src/docs/reference/About.js [hunk 2]
```diff
@@ -3,21 +3,131 @@
...
+        <Section>
+          <Paragraph size="large">
+            Grommet came to the world from four individuals inside 
+            Hewlett Packard that wanted to make designing a modern 
+            web experience for enterprise companies (or anyone really) 
+            easy. Working day to day, the designer and developer 
+            handoff is always a point of contention when it 
+            comes to our productivity and ensuring a 
+            unique vision is delivered to 
+            customers—So that’s where we started.
```
Comment (alansouzati) [inline src/docs/reference/About.js:63]: Can we add spacing here?

File: src/docs/reference/About.js [hunk 3]
```diff
@@ -3,21 +3,131 @@
...
+            for our own organization’s needs. We quickly 
+            found the usefullness for a tool like this outside 
+            Hewlett Packard, and taking a note from our own 
+            founders, Dave and Bill, we shared it with the 
+            open source community with great response. As 
+            with any company, change is inevitabe, 
+            and change it did…Hewlett Packard became 
+            two companies. The split became proof that G
+            rommet was not only a valuable tool 
+            to the community, but to our own eccosystem.
```
Comment (alansouzati) [inline src/docs/reference/About.js]: I think eccosystem should be ecosystem.

File: src/docs/reference/About.js [hunk 4]
```diff
@@ -3,21 +3,131 @@
...
+          </Paragraph>
+          <Paragraph size="medium">
+            Our goal is to continually enhance our 
+            toolset and make better expeirences easier to create.
```
Comment (alansouzati) [inline src/docs/reference/About.js]: experiences

File: src/docs/reference/About.js [hunk 5]
```diff
@@ -3,21 +3,131 @@
...
+            to form Grommet. Taking the knowldge from various 
+            web technologies and customer focused experiences, 
+            we ventured out to the community to bootstrap with 
+            the best tools out there. Initially using 
+            Facebook’s React.js, Node.js, and Inuit as the 
+            underlying technologies, and layering on a 
+            visual design system and with some secret sauce to 
+            tie it all together. This provided the 
+            flexibility needed to create a dynamic library of components and 
+            allow Grommet to be the leading React UI
```
Comment (alansouzati) [inline src/docs/reference/About.js]: can we say "one of the leading React UI .." :)

**Review by alansouzati — CHANGES_REQUESTED** [review]

File: src/docs/reference/About.js (hunk 3 above)
Comment (RyanCCollins) [reply src/docs/reference/About.js]: Oops, I had just copy and pasted this and didn't think to spell check.  I will do a spell check and resubmit this.

Comment (alansouzati) [reply src/docs/reference/About.js]: no problem man! thanks for looking into this in the first place.

**Comment by alansouzati** [comment]
Thanks!

**Comment by RyanCCollins** [comment]
Thank you! @alansouzati it looks like the images did not make it into the build :thinking:.  Sorry about that.  Let me see if I can figure out why.

**Comment by alansouzati** [comment]
yeah it is because the confusing structure I set up for docs. I've fixed it in here:

https://github.com/grommet/grommet-docs/commit/90e2f91033058d6c9b1c13a8bbdc6dbba0c3e1cf

**Comment by RyanCCollins** [comment]
Thank you sir!

**Pull request merged**